docs/Deploying/Configuration.md

```diff
@@ -1184,14 +1184,35 @@ osquery:
   status_log_plugin: firehose
   result_log_plugin: firehose
 ```
+#### External Activity Audit Logging
 
-#### activity_enable_audit_log
+> Applies only to Fleet Premium. Acitivity information is available for all Fleet instances using the [Activities API](https://fleetdm.com/docs/using-fleet/rest-api#activities).
+
+Stream Fleet user activities to logs using Fleet's logging plugins.
+
+The following types of activity are included:
+
+Created pack
+Edited pack
+Deleted pack
+Applied pack with fleetctl
+Created policy
+Edited policy
+Deleted policy
+Applied policy with fleetctl
+Created saved query
+Edited saved query
+Deleted saved query
+Applied query with fleetctl
+Ran live query
+Created team
+Deleted team
+
+##### activity_enable_audit_log
 
 This enables/disables the log output for audit events.
 See the `activity_audit_log_plugin` option below that specifies the logging destination.
 
-The audit events are logged in an asynchronous fashion. It can take up to 5 minutes for an event to be logged.
```

Comment:
Any reason to delete this? It's important to note the logging happens every 5 minutes.